hastexo_guacamole_client/settings.py

```diff
@@ -1,24 +1,19 @@
 import os
+from distutils.util import strtobool
+from yaml import load, SafeLoader
+from yaml.scanner import ScannerError
 
 from django.core.exceptions import ImproperlyConfigured
 
 
-def get_env_setting(setting):
-    """ Get the environment setting or return exception """
-    try:
-        return os.environ[setting]
-    except KeyError:
-        error_msg = u"Set the %s env variable" % setting
-        raise ImproperlyConfigured(error_msg)
-
-
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = get_env_setting('HASTEXO_GUACAMOLE_SECRET_KEY')
+SECRET_KEY = os.environ.get('HASTEXO_GUACAMOLE_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv('HASTEXO_GUACAMOLE_DEBUG', False)
+DEBUG = bool(strtobool(os.environ.get('HASTEXO_GUACAMOLE_DEBUG', False)))
```

Comment:
This won't work for an unset envar:

```doctest
>>> bool(strtobool(os.environ.get('HASTEXO_GUACAMOLE_DEBUG', False)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/distutils/util.py", line 313, in strtobool
    val = val.lower()
AttributeError: 'bool' object has no attribute 'lower'
```
The default value passed into `os.environ.get()` has to be `'false'` (string), not `False` (boolean).

Reply:
Done.